### DISCUSSION TITLE
Inline SVG (`<svg>`)

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#125] [State of HTML Features]
Embed SVG vector graphics directly in markup

### Code sample

```html
<svg>
  <circle cx="50" cy="50" r="50" />
</svg>
```

### MDN URL:

https://developer.mozilla.org/en-US/docs/Web/SVG/Element/svg

### Specification URL:

https://www.w3.org/TR/SVG2/struct.html#NewDocument

### How stable is this feature?

Stable (Widely supported for several years)

### Type of feature

HTML element

### What focus area does this feature fall into?

Embedding

### Details

We could also ask about individual SVG elements, e.g. `<use>`

**Comment by AnthonyLenglet** [comment]
What would be asked about this topic, just overall use of the `<svg>` element, or something more specific ?
This is an area where I feel like most people use this element, albeit mostly subconsciously (e.g. via the use of libraries like Font Awesome)

**Reply by LeaVerou** [reply]
See [my blog post](https://lea.verou.me/blog/2023/design-state-of-html/) for background about the Part 1 questions, they have standardized answers, though [quick context](https://github.com/Devographics/surveys/discussions/163) could help provide more context about the answers.

**Reply by AnthonyLenglet** [reply]
oh I see, that clears it up, thank you !

I don't know if there is a lot of people actually willingly using specific svg elements, since people usually make those assets using an external tool and export them as svg

I feel like if it would be better to ignore asking about individual svg elements and instead ask about wether or not people are using SVGs +  if they are importing svgs via the `img` tag or using `svg` (maybe there could also be something about the tools used to create the SVGs ? don't know if that would actually be helpful data though)
